README.md

```diff
@@ -97,7 +97,7 @@ Depending on the type of use you want like production of *Markdown* or maybe to
 
 ### And More
 
-* **i18n**: English, Dutch, Korean, Japanese, Chinese, Spanish, German, Russian, French, Ukrainian, Turkish, Finnish, Czech, Arabic, Polish, Galician, Swedish, Italian + language you extend.
+* **i18n**: English, Dutch, Korean, Japanese, Chinese, Spanish, German, Russian, French, Ukrainian, Turkish, Finnish, Czech, Arabic, Polish, Galician, Norwegian, Swedish, Italian + language you extend.
```

Comment:
Can you move it after _Italian_? This is not the alphabetical order, but the order in which languages ​​are added.
```
... Swedish, Italian, Norwegian + ...
```